portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,13 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Fetch a welcome message from the servlet, when clicked the button
```

Comment:
Per https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments, please write function comments in the third person.
```
/**
* Fetches ...
*/
```

Reply:
Fixed.